src/js/common/schemaform/fields/ArrayField.jsx

```diff
@@ -192,7 +192,7 @@ export default class ArrayField extends React.Component {
     const DescriptionField = typeof description === 'function'
       ? uiSchema['ui:description']
       : null;
-    const isReviewMode = uiSchema['ui:options'].reviewMode;
+    const isReviewMode = formData && formData.length;
```

Comment:
Can you remind me again what review mode does?

Reply:
I'm also not sure I understand why this change is being made.

Reply:
The 526 form will display the review form of the array field if data is available (prefilled), rather than the standard form view initially. 
![screen shot 2018-04-25 at 11 55 46 am](https://user-images.githubusercontent.com/16051172/39266706-a12f569a-487f-11e8-9244-ed054607f20b.png)
If we want to hold off on this for the rest of the forms for now, perhaps we should create a 526 specific array field?

Reply:
Yeah, I think this will have some negative effects on other forms, so we should come up with a different way of achieving it.

Off the top of my head, we might be able to add an option to ArrayField that starts with all items in the read only state if some condition is met.

Reply:
Sounds good, thanks @jbalboni, we will need to pass the formConfig via routes in order to have access to that form config option, I'll push up those changes for your consideration.

Reply:
We currently have it set up to specify this in uiSchema, but array field needs to be passed the formConfig from routes rather than from form in order to access that information. I'll make the changes to provide that access, and then update the reviewMode logic to also require that config setting.